nexus_constructor/json/transformation_reader.py

```diff
@@ -301,6 +301,19 @@ def _create_transformations(self, json_transformations: list):
             vector = self._find_attribute_in_list(
                 CommonAttrs.VECTOR, name, attributes, [0.0, 0.0, 0.0]
             )
+            offset_vector = self._find_attribute_in_list(
+                CommonAttrs.OFFSET, name, attributes, [0.0, 0.0, 0.0]
+            )
+
+            offset_units = self._find_attribute_in_list(
+                CommonAttrs.OFFSET_UNITS, name, attributes
+            )
+            if not offset_units:
+                if offset_vector is [0.0, 0.0, 0.0]:
+                    continue
+                else:
+                    offset_units = ""
+
```

Comment:
It's done like this because offset_vector could be supplied, meaning an offset_units is needed, and we should have a blank value until the user sets it (it will appear red in the UI).  If we pass a default value as in the suggestion it means other parts of the program will be triggered incorrectly.

offset_vector = 0,0,0, offset_units = None: do not include offsets in model
offset_vector = x,y,z, offset_units = None: set offset_units to an error value to be corrected by the user
offset_vector = x,y,z (including (0,0,0)), offset_units = "something" (including ""): we aren't doing anything with the offset but we are going to put it into our model, maybe we want it included in our JSON but not added in the Constructor for some reason